### PR TITLE
tests: filestore alert event

### PR DESCRIPTION
**Pull request by catenacyber**
Replaces #1058 with added README

### CODE REVIEW

**Comment by catenacyber** [comment]
Requires https://github.com/OISF/suricata/pull/8368

File: tests/filestore-alert-log/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Description
+
+Test alert event with filestore
+
+# PCAP
+
+The pcap comes from test filestore-filecontainer-http
```
Comment (inashivb) [inline tests/filestore-alert-log/README.md:7]: We can avoid copying the pcap file here by using the `pcap` feature in `test.yaml`. e.g. https://github.com/OISF/suricata-verify/pull/1128/files#diff-43a344381d7ddf9d8a334ece41d352d6a338438cc1bd465b54d24c64858c099dR1

**Review by inashivb — COMMENTED** [review]
Looking great. :)
I left an improvement suggestion using a feature I was recently reminded of.

File: tests/filestore-alert-log/README.md (hunk 1 above)
Comment (catenacyber) [reply tests/filestore-alert-log/README.md:7]: Nice, doing it

Is there a ticket for CI to catch new pcap being a duplicate ?

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/1146

File: tests/filestore-alert-log/README.md (hunk 1 above)
Comment (inashivb) [reply tests/filestore-alert-log/README.md:7]: I don't believe so.. I could make one.

**Pull request closed without merging**